ddtrace/__init__.py

```diff
@@ -1,4 +1,9 @@
-from ._logger import configure_ddtrace_logger
+from ddtrace.internal.module import ModuleWatchdog
+
+
+ModuleWatchdog.install()
```

Comment:
is this related?

this is needed now because we need it in order to set a callback for gevent loading?

should we enable `ModuleWatchdog` by default in a separate PR? seems "unrelated", or something that can be pulled out for clarity. wdyt?

Reply:
> this is needed now because we need it in order to set a callback for gevent loading?
Correct!
> should we enable `ModuleWatchdog` by default in a separate PR? seems "unrelated", or something that can be pulled out for clarity. wdyt?
Yes, we could do this.